showcase/src/app/demo/sidebar/demo-sidebar.component.html

```diff
@@ -0,0 +1,3 @@
+<example-viewer [extensions]="['HTML']" filesPath="sidebar/regular" exampleTitle="SHOWCASE.DEMO.SIDEBAR.TITLE">
+	<p translate>SHOWCASE.DEMO.SIDEBAR.INTRO</p>
```

Comment:
Could you please provide an example with calls to the sidebarService ? 
You could provide 3 examples, one for each sidebar.